src/tests/search/search.test.js

```diff
@@ -0,0 +1,144 @@
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import faker from 'faker';
+import app from '../../app';
+import { managerToken1, nonManagerToken, loggedInManager2 } from '../fixtures/managers.fixture';
+import createRequests from '../fixtures/request.fixture';
+import { createTrip, multiCitytrip } from '../fixtures/trip.fixture';
+
+chai.should();
+chai.use(chaiHttp);
+
+describe('Test searching requests:', () => {
+  let departuredate;
+  before(async () => {
+    await createRequests();
+    await createTrip();
+  });
+  it('Should return status code of 200 on with search results when manager search by requester with pagination', (done) => {
+    chai.request(app)
+      .get(`/api/search?name=${loggedInManager2.firstName}`)
+      .set('Authorization', managerToken1)
+      .end((err, res) => {
+        expect(res).to.have.status(200);
+        expect(res.body).to.have.property('data');
+        expect(res.body.data).to.have.property('rows');
+        expect(res.body.data.rows[0]).to.have.property('id');
+        expect(res.body.data.rows[0]).to.have.property('requesterId');
+        expect(res.body.data.rows[0]).to.have.property('tripId');
+        expect(res.body.data.rows[0]).to.have.property('status');
+        expect(res.body.data.rows[0]).to.have.property('lineManagerId');
+        done();
+      });
+  });
+  it('Should return status code of 201 on successful trip creation', (done) => {
```

Comment:
Why are you creating a trip here?

Reply:
I was creating a trip that later needs to be searched for. But maybe I can use the fixture instead